src/components/commons/layout/header/Header.tsx

```diff
@@ -1,9 +1,24 @@
 import * as S from "components/commons/layout/header/Header.styles";
+import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function Header() {
+  const [isDisplay, setIsDisplay] = useState(false);
+
+  const router = useRouter();
+
+  const onClickLogo = () => {
+    router.push("/");
+  };
+
+  const onClickProfileButton = () => {
+    if (!isDisplay) setIsDisplay(true);
+    else if (isDisplay) setIsDisplay(false);
+  };
+
```

Comment:
onClickProfileButton 함수 내부에 코드를 아래와 같이 구현해도 동일한 동작을 할 것으로 예상되는데 검토한번 요청드립니다! 
```suggestion
setIsDisplay(!isDisplay)
```